src/pages/api.astro

```diff
@@ -0,0 +1,30 @@
+---
+import Base from "@layouts/Base.astro";
+import swaggerCss from "swagger-ui/dist/swagger-ui.css?raw";
+import swaggerDarkCss from "../styles/swagger-dark.css?raw";
```

Comment:
Why not just
```js
import "swagger-ui/dist/swagger-ui.css"
import "../styles/swagger-dark.css"
```
?

Reply:
Most of the CSS gets optimized away because the build process believes it isn't used, as the html body is a result of a Javascript call on the client.

Reply:
Pain